tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs

```diff
@@ -95,6 +95,12 @@ public async Task AnEmptyListIsReturned()
         }
 
         [Then(@"the order with orderId (.*) is updated in the database with data")]
+        public async Task ThenTheOrderIsUpdatedInTheDatabase(string orderId, Table table)
+        {
+            var actual = await OrderEntity.FetchOrderByOrderId(_settings.ConnectionString, orderId);
+            table.CompareToInstance(actual);

```

Comment:
Interesting, not seen this approach before. For consistency reasons one suggestion would be to use the fluent assertions as these tend to provide more useful test failure messages.

Reply:
What would you recommend here bearing in mind that i only want to check the properties in the table and the table contains less properties than the entity.  Some of the entities properties are time stamps which will not match up on a table to properties comparison.?

Reply:
Here is the output 

TechTalk.SpecFlow.Assist.ComparisonException : The following fields did not match:
Description: Expected <This is an order for organisation 2X>, Actual <This is an order for organisation 2>, Using 'TechTalk.SpecFlow.Assist.ValueComparers.DefaultValueComparer'
OrderStatusId: Expected <2X>, Actual <2>, Using 'TechTalk.SpecFlow.Assist.ValueComparers.DefaultValueComparer'
OrganisationId: Expected <4af62b99-638c-4247-875e-965239cd0c48X>, Actual <4af62b99-638c-4247-875e-965239cd0c48>, Using 

If i don't do it this way  i think i will need to create poco objects and map data to them to and  the compare the objects with fluent assertions.